### DISCUSSION TITLE
v0.2.0 Initial release

### DISCUSSION THREAD

**Discussion by jkoehl** [jkoehl/homebridge-waterguru#2] [Announcements]
First release with water temperature support

<hr /><em>This discussion was created from the release <a href='https://github.com/jkoehl/homebridge-waterguru/releases/tag/v0.2.0'>v0.2.0 Initial release</a>.</em>

**Comment by PrayerfulDrop** [comment]
Any more recent updates to show PH and Chlorine yet?

**Comment by jkoehl** [comment]
No but I should get back at it since the season is upon us.On Mar 21, 2023, at 3:42 PM, Aaron Ward ***@***.***> wrote:﻿
Any more recent updates to show PH and Chlorine yet?

**Reply by PrayerfulDrop** [reply]
I can beta test with ya!

**Comment by dvohasek** [comment]
Hey All, I have some experience and lots of time off.  Just got the Sense2, let me know how i can help.  Would love to pull all the historical information.

**Comment by jkoehl** [comment]
Hi dvohasek,Feel free to put in a merge request.I did make some updates to show the Cl and Ph levels but there isn’t a true water chemistry type sensor defined in HomeKit so it only shows in apps that support showing proprietary custom attributes. And for history I think only the Eve app will show that.But hey if it helps folks that would benefit from those features then it’s worth a shot. I mostly wanted to be able to ask Siri what the pool temperature is and that has been working well at least.The WaterGuru API is undocumented but you can see in their App that it supports a decent amount of data.JohnOn Jun 19, 2023, at 4:18 PM, dvohasek ***@***.***> wrote:﻿
Hey All, I have some experience and lots of time off.  Just got the Sense2, let me know how i can help.  Would love to pull all the historical information.

**Comment by kofther** [comment]
Hey there.  Is this plugin still available?  I searched within the Homebridge UI and could not find it.  Thanks.

**Comment by jkoehl** [comment]
Hey did you try this again? I think NPM was down that day and it should work for you now.

**Comment by kofther** [comment]
Hey. I didn’t. I am away for the holidays and will try when I get back.  Thanks for the response.

Jim



On Dec 21, 2024, at 11:51 AM, John Koehl ***@***.***> wrote:


Hey did you try this again? I think NPM was down that day and it should work for you now.

**Comment by kofther** [comment]
Hey John.

I searched again and the plugin was not returned.

Jim

[Screenshot 2024-12-31 at 4.54.26 PM.png]


On Dec 21, 2024, at 8:51 AM, John Koehl ***@***.***> wrote:


Hey did you try this again? I think NPM was down that day and it should work for you now.

**Comment by kofther** [comment]
Sorry also meant to include this:


***@***.*** ~ % npm i -g homebridge-waterguru

npm ERR! code E404

npm ERR! 404 Not Found - GET https://registry.npmjs.org/homebridge-waterguru - Not found

npm ERR! 404

npm ERR! 404  'homebridge-waterguru@*' is not in this registry.

**Comment by jkoehl** [comment]
This is the package ***@***.***/homebridge-waterguruOn Dec 31, 2024, at 8:01 PM, kofther ***@***.***> wrote:﻿
Sorry also meant to include this:


***@***.*** ~ % npm i -g homebridge-waterguru

npm ERR! code E404

npm ERR! 404 Not Found - GET https://registry.npmjs.org/homebridge-waterguru - Not found

npm ERR! 404

npm ERR! 404  'homebridge-waterguru@*' is not in this registry.

**Comment by mzuniga51** [comment]
Hey.  Thanks of the effort with this plugin.  Any news on the Chlorine and PH data addition? Would be super interested. Thanks!
